src/kernel/chainstatemanager_opts.h

```diff
@@ -27,6 +27,7 @@ namespace kernel {
 struct ChainstateManagerOpts {
     const CChainParams& chainparams;
     const std::function<NodeClock::time_point()> adjusted_time_callback{nullptr};
+    const bool parallel_script_checks{false};
```

Comment:
In commit "Move ::g_parallel_script_checks into ChainstateManager" (fa39cf85d411e763d77f85d1e860e624e4a8e18b)

Not directly relevant to this PR (unless you want to add a todo comment), but since ChainstateManagerOpts is a kernel struct exposed to `bitcoin-chainstate.cpp`, probably it doesn't make sense to expose a boolean `parallel_script_checks` kernel option which is separate option than the number of script verification threads. Better to have one option and avoid inconsistencies.

Reply:
Yes, leaving as independent change, that can be done before or after the changes in this pull request

Reply:
nvm, reworked here